### PR TITLE
♿ Add initial keyboard navigation

### PR DESCRIPTION
**Pull request by victorborrasdev**
Adds the initial keyboard navigation through the days grid.

Next steps:
- Month and year navigation
- Trapping focus
- Testing on VUI

### CODE REVIEW

File: src/components/PickerDay.vue [hunk 1]
```diff
@@ -173,20 +192,68 @@ export default {
...
+      const newDate = new Date(this.focusedDate);
+      newDate.setDate(newDate.getDate() + 7);
+
+      const monthsDifference = newDate.getMonth() - new Date(this.pageTimestamp).getMonth();
+      const shouldChangePage = this.sideBySide ? monthsDifference > 1 : monthsDifference > 0;
+      if (shouldChangePage) {
+        this.changeMonth(1);
+      }
+
+      this.$emit('update:focusedDate', newDate.getTime());
```
Comment (ricardoferrolho) [inline src/components/PickerDay.vue]: This is repeated from `focusNextDay`. Let's keep it DRY.

File: src/components/PickerDay.vue [hunk 2]
```diff
@@ -173,20 +192,68 @@ export default {
...
+      const newDate = new Date(this.focusedDate);
+      newDate.setDate(newDate.getDate() - 7);
+
+      const monthsDifference = new Date(this.pageTimestamp).getMonth() - newDate.getMonth();
+      const shouldChangePage = monthsDifference > 0;
+      if (shouldChangePage) {
+        this.changeMonth(-1);
+      }
+
+      this.$emit('update:focusedDate', newDate.getTime());
```
Comment (ricardoferrolho) [inline src/components/PickerDay.vue]: Same here.

**Review by ricardoferrolho — CHANGES_REQUESTED** [review]

File: src/components/PickerDay.vue (hunk 1 above)
Comment (victorborrasdev) [reply src/components/PickerDay.vue]: I've refactored this and caught a bug (I hadn't taken into account jumping to a different year)

**Review by ricardoferrolho — APPROVED** [review]
:rocket:

**Pull request merged**